scripts/update-dockerhub-description.sh

```
@@ -0,0 +1,23 @@
+#!/bin/bash
+
```

we usually do
```
set -euxf -o pipefail
```
to assist in debugging and fail fast.